lib/elixir/pages/getting-started/writing-documentation.md

```diff
@@ -49,7 +64,7 @@ def size(%{size: size}) do
 end
 ```
 
-The compiler will infer this argument as `map`. Sometimes the inference will be suboptimal, especially if the function contains multiple clauses with the argument matching on different values each time. You can specify the proper names for documentation by declaring only the function head at any moment before the implementation:
+Sometimes the inference will be suboptimal, especially if the function contains multiple clauses with the argument matching on different values each time. You can specify the proper names for documentation by declaring only the function head at any moment before the implementation.
```

Comment:
Why did you remove the initial sentence?

Also please avoid small changes such as replacing `:` by `.`. It is very common to use `:` before examples in Elixir. If you change this here, this becomes inconsistent with the rest of the docs.

We need clear and focused changes. Thanks!